src/odemis/driver/pigcs.py

```diff
@@ -3539,16 +3541,16 @@ def sendOrderCommand(self, addr, com):
         else:
             full_com = "%d %s" % (addr, com)
         with self.ser_access:
-            logging.debug("Sending: '%s'", full_com.encode('string_escape'))
-            self.serial.write(full_com)
+            logging.debug("Sending: '%s'", full_com)
```

Comment:
We actually should escape `full_com`. Again, maybe use `%r` to avoid making it too complex.

Reply:
It seems to me that we don't support passing lists to an order command, only to query commands. Otherwise, the next line `serial.write(full_com)` would probably fail.